pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/segment/SegmentSizeBasedFlushThresholdUpdater.java

```diff
@@ -75,25 +74,14 @@ public void updateFlushThreshold(@Nonnull LLCRealtimeSegmentZKMetadata newSegmen
           "Committing segment size is not available, setting thresholds for segment {} from previous segment {}",
           newSegmentZKMetadata.getSegmentName(), committingSegmentZkMetadata.getSegmentName());
       newSegmentZKMetadata.setSizeThresholdToFlushSegment(committingSegmentZkMetadata.getSizeThresholdToFlushSegment());
-      newSegmentZKMetadata.setTimeThresholdToFlushSegment(committingSegmentZkMetadata.getTimeThresholdToFlushSegment());
       return;
     }
 
     long timeConsumed = System.currentTimeMillis() - committingSegmentZkMetadata.getCreationTime();
     int numRowsConsumed = (int) (committingSegmentZkMetadata.getTotalRawDocs());
     int numRowsThreshold = committingSegmentZkMetadata.getSizeThresholdToFlushSegment();
-    LOGGER.info(
-        "Time consumed:{} Time threshold:{} Num rows consumed:{} Num rows threshold:{} Committing segment size bytes:{}",
-        TimeUtils.convertMillisToPeriod(timeConsumed), MAX_TIME_THRESHOLD, numRowsConsumed, numRowsThreshold,
-        committingSegmentSizeBytes);
-
-    if (timeConsumed >= TimeUtils.convertPeriodToMillis(MAX_TIME_THRESHOLD)) {
-      LOGGER.info("Time threshold {} reached for committing segment {}", MAX_TIME_THRESHOLD,
-          committingSegmentZkMetadata.getSegmentName());
-    } else if (numRowsConsumed >= numRowsThreshold) {
-      LOGGER.info("Number of rows threshold {} reached for segment {}", numRowsThreshold,
-          committingSegmentZkMetadata.getSegmentName());
-    }
+    LOGGER.info("Time consumed:{}  Num rows consumed:{} Num rows threshold:{} Committing segment size bytes:{}",
+        TimeUtils.convertMillisToPeriod(timeConsumed), numRowsConsumed, numRowsThreshold, committingSegmentSizeBytes);
```

Comment:
The prevRatio should be the last calculated/applied ratio. You need to save it in the hash for that table so that you can use that next time a segment completes.

Reply:
Done.